oranda.json

```diff
@@ -1,6 +1,6 @@
 {
   "favicon": "https://www.axo.dev/favicon.ico",
-  "path_prefix": "oranda",
```

Comment:
we really gotta make sure to keep this out of the PRs...

Reply:
I know but fixing the path prefix broke my dev, fixed